src/common/net/http-client.ts

```diff
@@ -82,6 +87,18 @@ export abstract class HttpClient implements HttpClientInterface {
 
     return JSON.stringify(entity);
   }
+
+  private getSleepTime(retryAttempt: number): number {
```

Comment:
Same nit re: denoting time scale
```suggestion
  private getSleepTimeInMilliseconds(retryAttempt: number): number {
```